chainio/clients/eth/instrumented_client.go

```diff
@@ -42,6 +42,15 @@ func NewInstrumentedClient(rpcAddress string, rpcCallsCollector *rpccalls.Collec
 	}, nil
 }
 
+func NewInstrumentedClientFromClient(client *ethclient.Client, rpcCallsCollector *rpccalls.Collector) *InstrumentedClient {
```

Comment:
can you update `NewInstrumentedClient` to call this to prevent errors from duplicate code?